### PR TITLE
Refresh challenge score

### LINKED ISSUE #76: Make results per challenge
When watching challenge results see the top contributors for just that challenge and no other.

### PR DESCRIPTION
**Pull request by mariuscoto**
Fixed #76

### CODE REVIEW

**Comment by mariuscoto** [comment]
@maria please review

File: routes/challenge.js [hunk 1]
```diff
@@ -190,8 +190,12 @@ exports.edit = function(req, res) {
         'end':         new Date(req.body.end.replace(pattern, '$3-$2-$1'))
     }};
     Challenges.update(conditions, update, function (err, num) {
-      console.log("* Owner made changes to challenge " + req.body.name);
-      res.redirect('/challenges/' + req.body.name.replace(/\s+/g, '') + '/admin');
+      if (err) {
+        console.log("Failed to edit challenge name to: " + req.body.name)
```
Comment (maria) [inline routes/challenge.js]: Meh.. Logheaza erorile in... sistem nu in browser.

Comment (maria) [inline routes/challenge.js]: We can use https://github.com/jonnyreeves/js-logger#log-handler-functions -> and add authentication to the route :)

Comment (maria) [inline routes/challenge.js]: Or this one https://github.com/pimterry/loglevel -> you can save the output to a file

File: views/challenge.jade [hunk 2]
```diff
@@ -281,6 +283,14 @@ block content
 
       - if (currentUrl == '/challenges/' + challenge.link + '/results')
         h2 Results
+        
+        - if (challenge.status != 'closed')
+          .announcement
+            .announcement-inner    
+              | This is NOT visible to participants.
+              | Change Challenge status to "closed" to make it visible.
+          .spacer
```
Comment (maria) [inline views/challenge.jade]: I don't understand this part :)

File: routes/challenge.js (hunk 1 above)
Comment (maria) [inline routes/challenge.js]: opened #80

**Comment by maria** [comment]
if you know what you are doing you can merge this :)

**Pull request merged**